ql/src/java/org/apache/hadoop/hive/ql/exec/ExplainTask.java

```diff
@@ -1434,10 +1434,11 @@ public static String getExplainOutput(BaseSemanticAnalyzer sem, QueryPlan plan,
     PrintStream ps = new PrintStream(baos);
     try {
       List<Task<?>> rootTasks = sem.getAllRootTasks();
+      boolean isExtended;
```

Comment:
@ryukobayashi, i don't follow. Why can't we initialize it here?
````
boolean isExtended = conf.getBoolVar(ConfVars.HIVE_LOG_EXPLAIN_OUTPUT_INCLUDE_EXTENDED)
````
and then use it in both branches?